src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,189 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import TodoList from '../TodoList';
+import './TodoForm.css';
+
+export class TodoForm extends Component {
+  state = {
+    todos: [...this.props.todos],
+    name: '',
+    userId: 0,
+    title: '',
+    errorTitle: false,
+    errorPersone: false,
+    errorLength: false,
+    errorNotValid: false,
+  }
+
+  handleTitleChange = ({ target }) => {
+    if (target.value.length > 40) {
```

Comment:
It would be better if create separate variable for max available length

Reply:
completely agree